packages/compass-auto-updates/src/stores/store.js

```diff
@@ -19,8 +19,8 @@ store.onActivated = () => {
     /**
      * Listen to changes in user preferences.
      */
-    preferences.listenToAndRun(preferences, 'change:autoUpdates', () => {
-      if (preferences.isFeatureEnabled('autoUpdates')) {
+    preferences.onPreferenceChanged('autoUpdates', () => {
```

Comment:
Didn’t realize we were also doing this for auto-update, opened https://jira.mongodb.org/browse/COMPASS-6107 like the one for telemetry :+1: